### ISSUE TITLE
Function.extractParams is deprecated.

### ISSUE THREAD

**Issue by varemenos** [jakesankey/line-diff-details#33]
Assigning custom properties to a marker when creating/copying it is
deprecated. Please, consider storing the custom properties you need in
some other object in your package, keyed by the marker's id property.
```
Function.extractParams (/usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/marker.js:43:14)
MarkerLayer.markRange (/usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/marker-layer.js:196:71)
DisplayMarkerLayer.markBufferRange (/usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/display-marker-layer.js:90:62)
TextEditor.markBufferRange (/usr/share/atom/resources/app.asar/src/text-editor.js:1753:38)
LineDiffWorker.decorateDiffMarkers (/home/adonisk/.atom/packages/line-diff-details/lib/line-diff-worker.coffee:84:28)
HTMLDivElement.<anonymous> (/home/adonisk/.atom/packages/line-diff-details/lib/line-diff-worker.coffee:57:24)
```

**Comment by varemenos** [comment]
This is most probably due to Atom v1.13

**Comment by UziTech** [comment]
this should be fixed by pull request #34
